devices.h

```diff
@@ -34,3 +34,9 @@
 #define SC3	0xff	//Subclass
 #define PR3	0x00	//Protocol
 
+/* ALLVIEW P5 normal mode */
+#define VID4 0x0bb4  // Vendor id in ADB mode
```

Comment:
Could you use VID5, PID5... and send the request?  Also, correct the class, subclass and interface values.